extensionsGallery-insider.json

```diff
@@ -2961,6 +2961,62 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "61",
+					"extensionName": "Visualization",
+					"displayName": "Visualization",
+					"shortDescription": "Visualization of databases",
+					"publisher": {
+						"displayName": "Jens Hunt",
+						"publisherId": "R0tenur",
+						"publisherName": "R0tenur"
+					},
+					"versions": [
+						{
+							"version": "0.1.0",
+							"lastUpdated": "2/2/2020",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",
+									"source": "https://github.com/R0tenur/visualization/releases/tag/v0.1.0"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+									"source": "https://raw.githubusercontent.com/R0tenur/visualization/master/images/logo.png"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Content.Details",
+									"source": "https://github.com/R0tenur/visualization/blob/master/README.md"														},
```

Comment:
You need to use the raw content links for Details, Manifest and License

Reply:
this comment is also applicable to the other file you updated.

Reply:
I´ve updated the links in the files.